components/integrations/bots/bots.jsx

```diff
@@ -206,7 +206,7 @@ export default class Bots extends React.PureComponent {
                 helpText={
                     <FormattedMessage
                         id='bots.manage.help'
-                        defaultMessage='Use {botAccounts} to integrate with Mattermost through plugins or the API. Bot accounts are available to everyone on your server.'
+                        defaultMessage='Use {botAccounts} to integrate with Mattermost through plugins or the API. Bot accounts are available to everyone on your server. Enable bot account creation in the [System Console]({siteURL}/_redirect/integrations/bots).'
```

Comment:
@jfrerich wondering if you'd know why the redirect links render as plain text instead of a link? I've tried two approaches for redirects but each time it's hard coded. Same with`{botAccounts}` which is not a Markdown rendered link.

Is it because line 207 is `FormattedMessage` instead of `FormattedMarkdownMessage`?

![image](https://user-images.githubusercontent.com/13119842/64620614-38143a80-d3b2-11e9-84a9-09471c3452a1.png)
FormattedMarkdownMessage